css/style.css

```diff
@@ -0,0 +1,152 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 60%;
+}
+
+body{
+    font-family:'Times New Roman', Times, serif;
+    background-color: #2b2d42;
+}
+
+a{
+    text-decoration: none;
+    font-weight: 400;
+    /* 1.6*10 = 16p */
+    font-size: 2rem;
```

Comment:
defining the font size relative to root of percantage, I thin a fixed value like 16px will be better so 2rem = 32px for example